charts/dvpe-deployment-gloo/templates/gloo_authConfigPlugin.yaml

```diff
@@ -39,20 +39,22 @@ spec:
               {{- if .Values.gloo.authConfig.spec.configs.oauth.cookieDomain }}
               domain: {{ .Values.gloo.authConfig.spec.configs.oauth.cookieDomain | quote }}
               {{- end }}
-              maxAge: 0
+              maxAge: {{ .Values.gloo.authConfig.spec.configs.oauth.maxAge }}
+            {{- if .Values.gloo.authConfig.spec.configs.oauth.cache.enabled }}
+            redis:
+              cookieName: {{ .Values.gloo.authConfig.spec.configs.oauth.cache.cookieName }}
```

Comment:
if a cookie name other than` auth0-session` leads to an erroneous deployment, than you could also hard code it here.